pallets/Polkadex_Ido/src/test.rs

```diff
@@ -548,6 +593,16 @@ fn test_withdraw_token() {
             PolkadexIdo::withdraw_token(Origin::signed(4), round_id, 2),
             Error::<Test>::NotACreater
         );
+
+        //Test creator withdraw tokens during when fundraising is not close should return error : WithdrawalBlocked
+        system::Pallet::<Test>::set_block_number(closing_block_number - 1);
+        assert_noop!(
+            PolkadexIdo::withdraw_token(Origin::signed(ALICE), round_id, investor_address),
+            Error::<Test>::WithdrawalBlocked
+        );
+
+        //Test creator withdraw tokens during when fundraising is closed should be successful
+        system::Pallet::<Test>::set_block_number(closing_block_number + 1);
```

Comment:
Set this to closing_block_number to test corner case.

Reply:
OK